python/docs/additions/compute/indices_nonzero.rst

```diff
@@ -0,0 +1,28 @@
+indices_nonzero
+===============
+
+Returns
+-------
+
+pyarrow.lib.UInt64Array
```

Comment:
```suggestion
pyarrow.UInt64Array
```

(we can use the public import name, in which case it will also automatically link)